helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -10,41 +10,52 @@ spec:
   groups:
   - name: service-level
     rules:
-    - expr: "count(up{app='kubernetes'}) by (cluster_id)"
+    - expr: "count(up{app='kubernetes'}) by (cluster_type,cluster_id)"
       labels:
+        class: HIGH
         service: api-server
       record: raw_slo_requests
-    - expr: "sum((up{app='kubernetes'} * -1) + 1) by (cluster_id)"
+    - expr: "sum((up{app='kubernetes'} * -1) + 1) by (cluster_type,cluster_id)"
       labels:
+        class: HIGH
         service: api-server
       record: raw_slo_errors
     - expr: "kube_node_status_condition{condition='Ready'}"
       labels:
+        class: MEDIUM
         service: kubelet
       record: raw_slo_requests
-    - expr: "kube_node_status_condition{condition='Ready', status=~'false|unknown'}"
+    - expr: |
+        (
+          kube_node_status_condition{condition="Ready", status!="true"}
+          and
+          on (node) kube_node_spec_unschedulable == 0
+        )
+        and
+        on (node) time() - kube_node_created > 10 * 60
```

Comment:
Why is that?

Reply:
See https://github.com/giantswarm/g8s-prometheus/pull/1140

Reply:
Ok